dockstore-webservice/src/main/java/io/dockstore/webservice/resources/Description.java

```diff
@@ -34,6 +34,7 @@
         @Tag(name = "containers", description = "List and register entries in the dockstore (pairs of images + metadata (CWL and Dockerfile))"),
         @Tag(name = "containertags", description = "List and modify tags for containers"),
         @Tag(name = "GA4GH", description = "A curated subset of resources proposed as a common standard for tool repositories"),
+        @Tag(name = "extendedGA4GH", description = "An extension of GA4GH"),
```

Comment:
Maybe expand on the description. "Extended resources that use GA4GH tool registry schema objects but have not been formally proposed to the GA4GH yet"